clang/test/SemaSYCL/non-trivially-copyable-kernel-param.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsycl-new-kernel-param-requirements -verify -fsyntax-only %s
+
+// This test checks if compiler reports compilation error on an attempt to pass
+// a struct with non-trivially copyable type as SYCL kernel parameter.
+
+struct A { int i; };
+
+struct B {
+  int i,j;
```

Comment:
```suggestion
  int i;
```
What do we need `j` for?

Reply:
No need in `j`, removed